docs/pipeline-components-and-applications/stream-collector/configure/index.md

```diff
@@ -109,13 +109,13 @@ If no domain is specified, the cookie will be set against the full collector dom
 
 In earlier versions, you could specify a `domain` to tie the cookie to. For example, if set to `.snplow.com`, the cookie would have been accessible to other applications running on `*.snplow.com`. To do the same in this version, use the `fallbackDomain` setting but **make sure** that you no longer include a leading dot:
 
-```
+```properties
 fallbackDomain = "snplow.com"
 ```
 
 The cookie set by the collector can be treated differently by browsers, depending on whether it's considered to be a first-party or a third-party cookie. In earlier versions (0.15.0 and earlier), if you had two collector endpoints, one on `collector.snplow.com` and one on `collector.snplow.net`, you could only specify one of those domains in the configuration. That meant that you were only able to set a first-party cookie server-side on either `.snplow.com` or `.snplow.net`, but not on both. From version 0.16.0, you can specify a list of domains to be used for the cookie (**note the lack of a leading dot**):
 
-```
+```properties
```

Comment:
does not look like .properties to me

Reply:
Oh yeah, I remember these. Hocon doesn't seem supported in prism.

Hocon is supposed to be a json superset, if I do JSON I get:

<img width="661" alt="Screenshot 2022-09-14 at 22 30 23" src="https://user-images.githubusercontent.com/8820009/190266518-f61de297-c207-4518-a658-43a1ec815a9e.png">

Albiet incorrect, properties gave me:

<img width="633" alt="Screenshot 2022-09-14 at 22 30 33" src="https://user-images.githubusercontent.com/8820009/190266661-efc40f14-4759-41f9-903e-27c73cb0af2f.png">

Or I could pick `toml` which is reasonably similar to HOCON in style so does format ok...

<img width="681" alt="Screenshot 2022-09-14 at 22 36 32" src="https://user-images.githubusercontent.com/8820009/190267118-26ce7146-9d38-4f33-ba13-fd3c68619294.png">

Maybe json is the closest to a correct choice and easiest for everyone else to follow? I'll make that change, let me know if you have an opinion any other way.

Reply:
Just whipping through these and I notice some of the others only render with any colour if I pick `properties`.

<img width="406" alt="Screenshot 2022-09-14 at 22 45 01" src="https://user-images.githubusercontent.com/8820009/190268417-22262f17-16be-4d0f-abf9-8744e5fce527.png">

vs `json`
<img width="389" alt="Screenshot 2022-09-14 at 22 44 53" src="https://user-images.githubusercontent.com/8820009/190268420-ac00b9f4-fbda-454d-9104-dcb2f14c78a6.png">